salt/modules/win_status.py

```diff
@@ -539,21 +539,21 @@ def _win_remotes_on(port):
         remotes = set()
         try:
             data = subprocess.check_output(
-                ["netstat", "-n", "-p", "TCP"]
+                ["netstat", "-n"]
             )  # pylint: disable=minimum-python-version
         except subprocess.CalledProcessError:
             log.error("Failed netstat")
             raise
 
         lines = salt.utils.stringutils.to_unicode(data).split("\n")
         for line in lines:
-            if "ESTABLISHED" not in line:
+            if "ESTABLISHED" not in line or "TCP" not in line:
                 continue
             chunks = line.split()
             remote_host, remote_port = chunks[2].rsplit(":", 1)
             if int(remote_port) != port:
                 continue
-            remotes.add(remote_host)
+            remotes.add(remote_host.replace('[', '').replace(']', ''))
```

Comment:
**question** would it be a good idea to use the [`ipaddress` module](https://docs.python.org/3/library/ipaddress.html) here, or is that overkill?